### PR TITLE
Update to new build structure

### PR DESCRIPTION
**Pull request by thomas-bc**
Updating the tutorial so that it follows the new build structure of having the project() call at the top level and not in the deployment. Corrects the associated tutorial steps, as well as renaming MyComponents to Components as this is generated by default and skips the need to mess with CMake files.

### CODE REVIEW

File: docs/Deployments.md
Comment (hAkselS) [inline docs/Deployments.md:1]: Should we change MyDeployment to Deployment to match the standard?

File: docs/NewProject.md [hunk 1]
```diff
@@ -39,16 +39,28 @@ Select install_venv:
...
+3. `CMakeList.txt` and `project.cmake`: CMake files defining the build system
+4. `Components/`: directory to place user components in
+
 > Load the tools for this project via the virtual environment.
 > 
 > ```bash
-> cd MyProject
-> . venv/bin/activate
+> # In: MyProject
+> source venv/bin/activate
```
Comment (LeStarch) [inline docs/NewProject.md:63]: We should be careful when using `source` instead of `.` as `.` is ironically more cross-shell than source.

**Review by LeStarch — APPROVED** [review]

File: docs/NewProject.md (hunk 1 above)
Comment (thomas-bc) [reply docs/NewProject.md]: Done

File: docs/Deployments.md
Comment (thomas-bc) [reply docs/Deployments.md:1]: Changed to HelloWorldDeployment as we discussed

**Review by LeStarch — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Hello World: Introduction and F´ Terminology
 
-> Note: Tutorials can be browsed through our website or through the GitHub repository:
-> -> [Go to the website](https://fprime-community.github.io/fprime-tutorial-hello-world)
+> Note: Tutorials can be browsed through our website or through the GitHub repository:\
```
Comment (hAkselS) [inline README.md:3]: Is the \ after repository supposed to be there?

**Review by hAkselS — APPROVED** [review]
I found that one \ that may not supposed to be there. Other than that, it looks perfect to me!

File: README.md (hunk 2 above)
Comment (thomas-bc) [reply README.md:3]: It forces the linebreak to be rendered

**Pull request merged**